### ISSUE TITLE
Проверка Отсутствует обязательная проверка признака ОбменДанными.Загрузка в обработчике события "ПередЗаписью" работает не всегда корректно.

### ISSUE THREAD

**Issue by s-d-c2019** [1C-Company/v8-code-style#791]
Проверка _Отсутствует обязательная проверка признака ОбменДанными.Загрузка в обработчике события "ПередЗаписью"_ (аналогично и другие события "ПриЗаписи", и т.п....) работает не всегда корректно.

Следующий код Перед записью будет признан ошибочным:

Процедура ПередЗаписью()
	
	Если Не ОбменДанными.Загрузка Тогда
		ОбработчикПередЗаписью();
	КонецЕсли;
	
КонецПроцедуры

Если в обработчике используется _ОбменДанными.Загрузка_ в любом из условий _Если_, считать, что проверка пройдена и ошибку не выдавать.

![image](https://user-images.githubusercontent.com/54451084/134556790-d60f3a6c-c4ee-4de8-881d-f55ebd201c36.png)

 - OS: Windows
 - Версия 1C:Code-Style-V8 0.1.0.119
 - Версия 1C:EDT Ruby 2021.2.5

**Comment by marmyshev** [comment]
Я бы сказал, что код в таком стиле - это спагетти код (это же не нужно доказывать, что это плохой код?). Неплохо бы еще понимать зачем такой код писать? 

Такой вот код - это тоже спагетти код - и тоже плохой паттерн. Хоть и проверка будет формально проходить.

```bsl
Процедура ПередЗаписью()

   Если ОбменДанными.Загрузка Тогда
	Возврат;
   КонецЕсли;

   ОбработчикПередЗаписью();
КонецПроцедуры
```

**Comment by s-d-c2019** [comment]
Согласен, ни один из вариантов не отличается красотой. Возвраты то же не нравятся. Помещать код обработки в такое условие - совсем плохой вариант. С чем-то приходится мириться.
